frontend/src/bundles/common/pages/budgets/budgets.tsx

```diff
@@ -50,17 +53,25 @@ const Budgets: React.FC<Properties> = ({ budgetCards }) => {
                                 date={card.date}
                             />
                         ))}
-                        <Button
+                        <div
```

Comment:
Why 'button' is replaced by 'div'?

Reply:
there was an error: button cannot be a button's parent